internal/exec/vendor_utils.go

```
@@ -608,3 +621,19 @@
 		return false, nil
 	}
 }
+
+func validateURI(uri string) error {
```

golangci-lint (1.62.2)

func `validateURI` is unused

(unused)